client/src/app/(onboarding)/login/page.tsx

```diff
@@ -1,3 +1,195 @@
+"use client";
+import "./login.css";
+import { Button, Typography, Divider, TextField, Link } from "@mui/material";
+import { useState } from "react";
+
 export default function Page() {
-  return <h1>Login Placeholder</h1>;
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [error, setError] = useState(false);
+  const [emailHelperText, setEmailHelperText] = useState("");
+  const [passwordHelperText, setPasswordHelperText] = useState("");
+
+  // Dummy Function, will handle login logic
+  const handleSubmit = () => {
+    console.log("Email:", email);
+    console.log("Password:", password);
+  };
+
+  const validateInput = () => {
+    setError(false);
+    setEmailHelperText("");
+    setPasswordHelperText("");
+
+    let hasError = false;
+
+    if (email === "") {
+      setError(true);
+      setEmailHelperText("Email is required");
+      hasError = true;
+    }
+
+    if (password === "") {
+      setError(true);
+      setPasswordHelperText("Password is required");
+      hasError = true;
+    }
+
+    if (hasError === false) {
+      handleSubmit();
+    }
+  };
+
+  return (
+    <div className="login-container">
+      <Typography variant="h2" sx={{ marginBottom: "10px" }}>
+        Welcome Back
+      </Typography>
+      <Button
+        className="border-line"
```

Comment:
if you're trying to access the .borderline classname from the global.css, it's "borderline" not "border-line"

Reply:
copy and pasted the button from howie. Did not mean to be using that class

Reply:
yes